modules/ROOT/examples/live-demos/url-conversion-none/index.html

```diff
@@ -0,0 +1,4 @@
+<textarea id="url-conversion-none">
+  <p>Some relative urls: <a href="https://www.tiny.cloud/docs/demo">content</a>. <img src="{{logofordemos}}" alt="Tiny Logo" width="128" height="128" /></p>
```

Comment:
this should be a relative link as per example?

closest URL without an index page
/tinymce/6/basic-example.html